doc/source/cluster-launcher-usage.rst

```diff
@@ -0,0 +1,193 @@
+.. _launcher-usage:
+
+Launcher Usage Guide
```

Comment:
A "Launcher" is a bit imprecise. Can we call this "Launching Clusters" or something?

Reply:
it's a bit like a package reference though (covering main commands)

Reply:
renamed to 'cluster launcher usage guide'